### ISSUE TITLE
Rich editor missing a title attribute on the embed menu.

### ISSUE THREAD

**Issue by tburry** [vanilla/vanilla#10839]
When I hover the buttons on the rich editor's embed menu I get tooltips for everything except the embed button. It appears as though it's lacking a title tag.

![New_Discussion_—_T1.png](https://images.zenhubusercontent.com/57087d1abf93871ccd991f73/5f585614-16ec-4689-979a-68c343a81eb5)
